Client/YooChooseNotifier.php

```diff
@@ -95,6 +101,18 @@ public function deleteContent($contentId)
     }
 
     /**
+     * Gets ContentType ID based on $contentId
+     *
+     * @param int $contentId
+     *
+     * @return int
+     */
+    protected function getContentTypeId($contentId)
+    {
+        return $this->contentService->loadContent($contentId)->contentInfo->contentTypeId;
```

Comment:
Prefer `loadContentInfo()`, less resource intensive than `loadContent()`, and you only need data from the ContentInfo object.